src/components/SuggestedItem/SuggestedItem.css

```diff
@@ -0,0 +1,91 @@
+.item {
+  box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.2),
+    0 1.5px 3.33px 0 rgba(0, 0, 0, 0.19);
+  width: 60px;
+  height: 102px;
+}
+
+.item--border--packed {
+  border:1px solid #4896c9ff;
+}
+
+.item--border--unpacked {
```

Comment:
Better CSS variable names can be item--border--white, being more specific about the styling it's applying to.

Reply:
This is what I renamed them to
```css
.item--border--white {}
.item--border--blue {}
```